apps/client/src/common/components/dialog/Dialog.module.scss

```
@@ -1,12 +1,16 @@
 .dialog {
   position: fixed;
-  top: 10%;
+  top: 1rem;
   left: 50%;
 
   transform: translateX(-50%);
```

Stylelint (17.14.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)